### PR TITLE
DL-6089, integrated timescales update and use plus sync from viewer

### PR DESCRIPTION
**Pull request by cmrhmrc**

### CODE REVIEW

File: app/services/TimescalesService.scala [hunk 1]
```diff
@@ -18,27 +18,48 @@ package services
...
 @Singleton
 class TimescalesService @Inject() (repository: TimescalesRepository, appConfig: AppConfig) {
+  val logger: Logger = Logger(getClass)
+
   def save(json: JsValue): Future[RequestOutcome[Unit]] =
     json.validate[Map[String, Int]].fold(_ => Future.successful(Left(ValidationError)), _ =>
       repository.save(json).map{
         case Left(_) => Left(InternalServerError)
-        case result => Right(())
+        case result =>
```
Comment (paulanderson1234) [inline app/services/TimescalesService.scala:38]: Showing as declaration never used ?

File: test/mocks/MockArchiveRepository.scala [hunk 2]
```diff
@@ -35,9 +34,9 @@ trait MockArchiveRepository extends MockFactory {
         .expects(id, user, processCode, process)
     }
 
-    def getByProcessCode(processCode: String): CallHandler[Future[List[JsObject]]] = {
-      (mockArchiveRepository.getByProcessCode(_: String)).expects(processCode)
-    }
+    // def getByProcessCode(processCode: String): CallHandler[Future[List[JsObject]]] = {
```
Comment (paulanderson1234) [inline test/mocks/MockArchiveRepository.scala:37]: Keeping for a reason ?

File: test/mocks/MockTimescalesService.scala [hunk 3]
```diff
@@ -34,8 +34,12 @@ trait MockTimescalesService extends MockFactory {
         .save(_: JsValue))
         .expects(timescales)
 
-    def get(): Future[RequestOutcome[JsValue]] =
+  def updateTimescaleTable(js: JsObject): CallHandler[Future[RequestOutcome[JsObject]]] =
       (mockTimescalesService
-        .get())
+        .updateTimescaleTable(_: JsObject))
+        .expects(js)
+    // def get(): Future[RequestOutcome[JsValue]] =
```
Comment (paulanderson1234) [inline test/mocks/MockTimescalesService.scala:41]: As above ? Keeping for a specific reason

File: app/services/TimescalesService.scala (hunk 1 above)
Comment (cmrhmrc) [reply app/services/TimescalesService.scala:38]: replaced with _

File: test/mocks/MockArchiveRepository.scala (hunk 2 above)
Comment (cmrhmrc) [reply test/mocks/MockArchiveRepository.scala:37]: removed

File: test/mocks/MockTimescalesService.scala (hunk 3 above)
Comment (cmrhmrc) [reply test/mocks/MockTimescalesService.scala:41]: removed

**Pull request merged**